mask_to_wkt/README.md

```diff
@@ -1,6 +1,6 @@
 # Mask Extraction
 
-![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay")
+![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay") 
```

Comment:
Guessing this was an unintentional change?
```suggestion
![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay")
```

Reply:
yep, unintentional